hack/cluster-clean.sh

```diff
@@ -30,10 +30,10 @@ function patch_remove_finalizers() {
 }
 
 function delete_kubevirt_cr() {
-    # Delete KubeVirt CR, timeout after 10 seconds
+    # Delete KubeVirt CR, timeout after 30 seconds
     set +e
-    _kubectl -n ${namespace} delete kv kubevirt --timeout=10s --ignore-not-found
-    _kubectl -n cdi delete service cdi-uploadproxy-nodeport
+    _kubectl -n ${namespace} delete kv kubevirt --timeout=30s --ignore-not-found || exit 0
```

Comment:
Could you explain what this exactly does now? It looks to me like we would just abort `delete_kubevirt_cr` and never remove the finalizers. Wouldn't this then potentially lead to hanging resources if something is not right?

Reply:
Thanks @rmohr, you are absolutely right, it was a copy and paste error, sorry....
The right command must be `|| true` and not `|| exit 0`

Reply:
With `set +e` errors will occur, but the script will keep running. So `make cluster clean` doesn't fail, but we can see the error in the log.

But since we force delete the objects, it will be nice if we can hide the timeout error. Otherwise, when looking at the job's log, the first message is this failure, even though the job succeded.